frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -223,6 +223,7 @@ const ProjectListView: React.FC<IProps> = ({ filterable, title, mode }) => {
   const handleDelete = async () => {
     const project = data?.find(p => p.projectNumber === deleteId);
     if (project) {
+      project.status = projectStatuses.find((x: any) => x.name === project.status)!;
```

Comment:
what is this doing?

Reply:
Are you just loading the status into context here?

Reply:
Yeah it was throwing an error as it only had the status.name before for some reason